glusterd2/commands/volumes/volume-expand.go

```diff
@@ -47,6 +48,26 @@ func volumeExpandHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	volinfo, err := volume.GetVolume(volname)
+	if err != nil {
+		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, err)
+		return
+	}
+
+	for index := range volinfo.Subvols {
+		for _, brick := range volinfo.Subvols[index].Bricks {
+
+			for _, b := range req.Bricks {
```

Comment:
duplicate entries need to be checked within `req.Bricks` also.

Reply:
@aravindavk added check